ui/job-view/JobView.jsx

```diff
@@ -42,10 +43,12 @@ JobView.propTypes = {
   selectedJob: PropTypes.object,
 };
 
+// Sometime of these props are not ready by the time this renders, so
+// need some defaults for them.
 JobView.defaultProps = {
   revision: null,
-  currentRepo: {},
   selectedJob: null,
+  currentRepo: {},
```

Comment:
The ordering of the `defaultProps` here differs from that in `propTypes`. Should this line change be dropped to preserve the existing ordering?

Reply:
ahh, yes.  Good call.